src/components/AddPlaidBankAccount.js

```diff
@@ -163,12 +164,12 @@ class AddPlaidBankAccount extends React.Component {
                     <PlaidLink
                         token={this.props.plaidLinkToken}
                         onSuccess={({publicToken, metadata}) => {
-                            console.debug('[PlaidLink] Success: ', {publicToken, metadata});
+                            Log.info('[PlaidLink] Success: ', false, {publicToken, metadata});
```

Comment:
Is it safe to log plaid tokens or metadata? I was mostly using this on dev to debug Plaid so maybe we want to change this to something more generic. I'm not sure we need to know anything other than it succeeded.

Reply:
Removed this and now we're just logging success.